data/fixtures/test_data.py

```diff
@@ -465,8 +465,10 @@ def __init__(self):
             self.student_answers.append(answer)
             # add a TA and Instructor answer
             answer = self.create_answer(assignment, self.get_authorized_ta())
```

Comment:
should add a new answer for both instructor and ta that is comparable (to help testing throughout a variety of situations). will require updating a lot of tests